code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -592,6 +592,29 @@
 	cover.name = name
 	cover.desc = desc
 
+/obj/machinery/porta_turret/centcomm_shuttle

```

Comment:
Either i am doing this wrong or its the assess_perp being weird because i moved it under the syndicate turret removed the stuff that was unneeded as it was already defined and it wont shoot aliens now. This is what i have so far http://imgur.com/a/FKvAT